UIKitTemplate/UIKitTemplate/Views/ChooseAdditivesView.swift

```diff
@@ -0,0 +1,140 @@
+// ChooseAdditivesView.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Протокол делегата экрана выбора добавок в кофе
+protocol ChooseAdditivesDelegate: AnyObject {
+    func closeView()
+    func chooseAddition(addition: String, isAdd: Bool)
+}
+
+/// Вью экана выбора добавок в кофе
+class ChooseAdditivesView: UIView {
+    // MARK: - Constants
+
+    private enum Constants {
+        /// Расстояние между двумя добавками
+        static let distanceBetweenAdditives: CGFloat = 50
+        /// Текст для лейбла выбора дополнительных элементов
+        static let chooseRoastLabelText = "Выберите дополнительные \nингредiенты"
+    }
+
+    // MARK: - Visual Components
+
+    /// Надпись "Выберите дополнительные ингредиенты"
+    private let chooseRoastLabel = {
+        let label = UILabel(frame: CGRect(x: 40, y: 53, width: 295, height: 44))
+        label.text = Constants.chooseRoastLabelText
+        label.font = .init(name: "Verdana-Bold", size: 18)
+        label.textAlignment = .center
+        label.numberOfLines = 2
+        return label
+    }()
+
+    /// Кнопка закрытия вью
+    private let closeViewButton: UIButton = {
+        let button = UIButton(frame: CGRect(x: 20, y: 26, width: 14, height: 14))
+        button.setImage(UIImage(systemName: "xmark"), for: .normal)
+        button.tintColor = .black
+        return button
+    }()
+
+    // MARK: - Public Properties
+
+    /// Ссылка на делегата
+    weak var delegate: ChooseAdditivesDelegate?
+
+    // MARK: - Initializers
+
+    init(frame: CGRect, choosenAdditives: [String]) {
+        super.init(frame: frame)
+        createView()
+        createViewWithAdditives(choosenAdditives: choosenAdditives)
+    }
+
+    @available(*, unavailable)
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    // MARK: - Private Methods
+
+    /// Метод установки таргетов для элементов UIControl
+    private func setTergetsForUIControl() {
+        closeViewButton.addTarget(self, action: #selector(closeViewButtonAction), for: .touchUpInside)
```

Comment:
```suggestion
        closeViewButton.addTarget(self, action: #selector(closeViewButtonAction), for: .touchUpInside)
```
в самих кнопках это создавай